packages/subgraph/manifest/subgraph.placeholder.yaml

```diff
@@ -159,14 +159,14 @@ templates:
           handler: handleProposalExecuted
         - event: ProposalCreated(indexed uint256,indexed address,bytes)
           handler: handleProposalCreated
-        - event: VoteSettingsUpdated(uint64,uint64,uint64,uint256)
-          handler: handleVoteSettingsUpdated
-  # Addresslist (package)
-  - name: Addresslist
+        - event: VotingSettingsUpdated(uint8,uint64,uint64,uint64,uint256)
+          handler: handleVotingSettingsUpdated
+  # AddresslistVoting (package)
+  - name: AddresslistVoting
     kind: ethereum/contract
     network: {{network}}
     source:
-      abi: Addresslist
+      abi: AddresslistVoting
```

Comment:
there is another `addresslist` remaining to renamed to  `AddresslistVoting``at line 74

Reply:
Fixed in 39717dc74e9628712b30d8ef04f90b9dff8b455d